push/services/SendWebPushService.py

```diff
@@ -0,0 +1,47 @@
+import json
+from pywebpush import webpush, WebPushException
+from push.models import Notification
+from pushNotification.settings import PRIVATE_KEY
+from push.services.SubscriptionService import SubscriptionService
+
+
+class SendWebPushService:
+    def __init__(self, notification_id):
+        self.notification_object = Notification.objects.filter(id=notification_id).first()
+
+    @staticmethod
+    def make_request_subscription_info(endpoint, public_key, auth_key):
```

Comment:
You can also just send the subscription object here and this method would return the subscription info

Reply:
use prefix like create, build instead of make